vtex-apps/avatax-br-admin
# adding request download button in log details

#### What problem is this solving?

Implementing a "Download Request" button in the log details so it's easier to download the request payload sent to Avalara for simulations/tickets.

#### How to test it?
Click on the log details button and then on the "Download Request" button inside the popup. The system should download a file containing the request payload sent to Avalara for that specific calculation.

[Workspace](https://avatax--sandboxbrdevbrl.myvtex.com/admin/avatax-br)

#### Screenshots or example usage:

<img width="936" alt="Screen Shot 2021-07-08 at 17 56 26" src="https://user-images.githubusercontent.com/1629129/124989805-eb353580-e015-11eb-8e87-a57213d73909.png">
